Contributing.md

```diff
@@ -0,0 +1,61 @@
+Contributor Guidelines
+===
+
+## Code Style
+* Line endings
+    * Use Unix line endings when committing (\n).
+    * Windows users of git can do `git config --global core.autocrlf true` to
+    let git convert them automatically.
+
+* Column width
+    * 80 for Javadocs
+    * 150 for code
+    * Feel free to wrap when it will help with readability
+
+* Indentation
+    * Use 4 space tabs for indentations, do not use spaces.
+
+* Vertical whitespace
+    * Place a blank line before the first member of a class, interface, enum,
+    etc. (i.e. after class Example {) as well as after the last member.
+
+* File headers
+    * File headers must contain the license headers for the project.
+    * You can use the `gradle licenseFormat` to automatically to do this for
+    you.
+
+* Imports
+    * Imports must be grouped in the following order:
+        * normal imports
+        * java imports
+        * javax imports
+        * static imports
+
+* Javadocs
+    * Do not use @author
+    * Wrap additional paragraphs in `<p>` and `</p>`
+    * Capitalize the first letter in the descriptions within each “at clause”,
+    i.e. @param name Player to affect, no periods
+    * Be descriptive when explaining the purpose of the class, interface,
+    enum, method etc.
+
+* Deprecation
+    * Do not deprecate content unless you have a replacement, or if the
+    provided feature is removed.
+    * When deprecating, provide the month and year when it was deprecated.
+    * Anything deprecated is kept for one major version.
```

Comment:
Not necessarily. A major version will remove deprecated features, but we'll keep them around in minor versions beforehand.